### ISSUE TITLE
Can't load schema when the DB already exists

### ISSUE THREAD

**Issue by mattgibson** [zilverline/sequent#137]
For our production deploy, the database already exists, but is empty. I can't run the `sequent:migrate:online` task as I get an error that the `event_records` table does not exist. Dropping the DB and running `sequent:db:create` is not an option as the app does not have permission to destroy the DB.

It would be ideal to have a `sequent:schema:create` task that assume the DB is already present.

**Comment by mattgibson** [comment]
Fixed manually in the console like so:

```
Sequent::Support::Database.create_schema(Sequent.configuration.event_store_schema_name)
sequent_schema = File.join(Sequent.configuration.database_config_directory, "#{Sequent.configuration.event_store_schema_name}.rb")
load(sequent_schema) 
```

**Comment by lvonk** [comment]
Yes that is indeed the fix. I have created rake task for this which will be available in master shortly.

**Comment by mattgibson** [comment]
Thanks @lvonk !
